packages/shorebird_cli/lib/src/commands/release/shorebird_release_ios_command.dart

```diff
@@ -0,0 +1,122 @@
+import 'dart:io';
+
+import 'package:collection/collection.dart';
+import 'package:mason_logger/mason_logger.dart';
+import 'package:shorebird_cli/src/auth_logger_mixin.dart';
+import 'package:shorebird_cli/src/command.dart';
+import 'package:shorebird_cli/src/config/config.dart';
+import 'package:shorebird_cli/src/shorebird_build_mixin.dart';
+import 'package:shorebird_cli/src/shorebird_config_mixin.dart';
+import 'package:shorebird_cli/src/shorebird_validation_mixin.dart';
+import 'package:shorebird_code_push_client/shorebird_code_push_client.dart';
+
+/// {@template release_ios_command}
+/// `shorebird release ios`
+/// Create new app releases for iOS.
+/// {@endtemplate}
+class ReleaseIosCommand extends ShorebirdCommand
+    with
+        AuthLoggerMixin,
+        ShorebirdValidationMixin,
+        ShorebirdConfigMixin,
+        ShorebirdBuildMixin {
+  /// {@macro release_ios_command}
+  ReleaseIosCommand({
+    required super.logger,
+    super.auth,
+    super.buildCodePushClient,
+    super.cache,
+    super.validators,
```

Comment:
Apologies if I'm repeating myself on this, but we'll need to update how we provide validators, since the only default validator is the AndroidInternetPermission validator, which doesn't apply here (and we'll likely want something similar to that, but for iOS).

Reply:
Yup sg will address in a separate PR 👍